pilot/pkg/networking/core/v1alpha3/gateway_test.go

```diff
@@ -678,6 +679,56 @@ func TestBuildGatewayListenerTlsContext(t *testing.T) {
 				RequireClientCertificate: proto.BoolFalse,
 			},
 		},
+		{
+			name: "ecdh curves specified in mesh config with tls SIMPLE",
```

Comment:
Same here. Can you add a test case for MUTUAL?

Reply:
Added additional test cases